src/metalkit.cs

```diff
@@ -39,6 +39,9 @@ interface MTKModel {
 #endif
 	interface MTKView : NSCoding, CALayerDelegate {
 
+		[Export ("init")]
+		IntPtr Constructor ();
```

Comment:
It's added by default. If not present it's because someone added `[DisableDefaultCtor]` (see above) and that's the line that should be removed.

We'll need to `git blame` why this was added before merging.

Reply:
The fact that it can be created does not mean it is usable per se, have you tried to do more than just `MTKView* t = [[MTKView alloc] init];`?

Reply:
I'll try it, but I think it works... I know I did create a view with this way when testing. Maybe I was mistaken. I'll confirm.

Reply:
There's a `[NullAllowed]` on the `IMTLDevice` device of the designated initializer and you can, in theory, give it a `CGRect.Empty` so there's nothing (again in theory) that should block the default `init` from working. Of course a test to double check theory means being sure in practice 😸

Reply:
git blame: https://github.com/xamarin/maccore/commit/5b9ab25cf8d807d5c2d8c92c424cb0b12875dfb5#diff-2c98c959c61fe409d8a9d47f642fcc230c3c2a37c760d18a80934cc46658e760R14

that didn't explain much, so we can try just removing the `[DisableDefaultCtor]` and see if the tests complain somewhere.